### ISSUE TITLE
Please consider keeping the supported vscode engine in package.json as low as possible to have bigger audience

### ISSUE THREAD

**Issue by rbadapanda** [robole/vscode-markdown-shortcuts#1]
Hello,
Thanks for the great extension.
I use Vscode 1.47. and this extension works on it.

But when I install it from the marketplace it says: unsupported vs version. it is because in package.json the minimum vs engine chosen is 1.48.

If you could keep it low, more version of vscode can use it.

This comment applies to your other extension in the pack - vscode-markdown-snippets as well.

Please consider.

**Comment by robole** [comment]
Hi,

Thanks! 🙏

I will look into it and see if it can be lowered. 👍

I looked at it briefly for a previous extension I made, when I tried to change the 'minimum vs engine' to an earlier version, visual studio marketplace refused it when I tried to upload it! The problem with the ecosystem is that the yeoman extension generator defaults to the latest version and the docs have no versioning info to get an idea about compatibility. I might just try to remove the field and see what happens!
